actionpack/lib/action_controller/metal/rendering.rb

```diff
@@ -160,6 +160,12 @@ def inherited(klass)
     #         render "posts/new", status: :unprocessable_entity
     #         # => renders app/views/posts/new.html.erb with HTTP status code 422
     #
+    # `:variants`
+    # :  This tells Rails to look for template variations of the same format.
```

Comment:
`format` might be a bit confusing as it is also used to describe `html` in this case.
```suggestion
    # :  This tells Rails to look for template matching the variations.
```

I'm not 100% sure the following is correct but it might be good to describe how this works when more that one variant is passed:
```suggestion
    # :  This tells Rails to look for the first template matching any of the variations.
```